tests/ui/delegation/glob-non-impl.stderr

```diff
@@ -0,0 +1,28 @@
+error: delegation is not supported in `extern` blocks
+  --> $DIR/glob-non-impl.rs:15:5
+   |
+LL |     reuse Trait::*;
+   |     ^^^^^^^^^^^^^^^
+   |
+   = help: consider moving the delegation out to a nearby module scope
```

Comment:
(well, that wouldn't work either)

Reply:
Hmm, I'll look if it's possible to disable this help for delegations specifically.